core/src/impl/Kokkos_TaskQueue.hpp

```diff
@@ -544,7 +544,7 @@ class TaskBase
   // Constructor for runnable task
   KOKKOS_INLINE_FUNCTION constexpr
   TaskBase( FunctorType && arg_functor )
-    : root_type() , functor_type( std::move(arg_functor) ) {}
```

Comment:
Why was this `std::move` removed?

Reply:
Function is not marked with '__host__ __device__' for CUDA.

Reply:
Did this actually cause compiler errors/warnings? I recently put `std::move` in to the `View` move constructors, and NVCC seemed to compile it okay. I also don't know of a way to replicate `std::move`'s behavior without calling it, its pretty central to the language.